README.md

```diff
@@ -20,6 +20,7 @@ This repository defines the following SAP vocabularies:
 - [PersonalData: Personal data / GDPR](vocabularies/PersonalData.md)
 - [Session: Sticky Sessions](vocabularies/Session.md)
 - [UI: Representing data in user interfaces](vocabularies/UI.md)
+- [Offline: Terms for Offline OData](vocabularies/Offline.md) - _experimental_
```

Comment:
Please move this new line up after the `ODM` line (line 18)

Reply:
Done, thanks a lot.